purchase-listinator.iml

```diff
@@ -34,7 +35,9 @@
     <orderEntry type="library" name="Leiningen: com.google.guava/guava:18.0" level="project" />
     <orderEntry type="library" name="Leiningen: com.googlecode.json-simple/json-simple:1.1.1" level="project" />
     <orderEntry type="library" name="Leiningen: com.h2database/h2:1.3.171" level="project" />
+    <orderEntry type="library" name="Leiningen: com.novemberain/langohr:5.1.0" level="project" />
```

Comment:
I think we should not commit this `.idea` directory. What do you think?

Reply:
I think you are right, I removed all files and add .idea folder in the gitignore